jenkins/bootstrap.py

```diff
@@ -735,11 +737,16 @@ def job_args(args):
     return [os.path.expandvars(a) for a in args]
 
 
-def job_script(job, use_json):
+def job_script(job, use_json, jobs_dir):
     """Return path to script for job."""
+    jobs_dir_resolver = lambda f: test_infra('jobs/%s' % f)
+
+    if jobs_dir:
+        jobs_dir_resolver = lambda f: os.path.join(jobs_dir, f)
+
     if not use_json:
```

Comment:
Right now, because we needed those flags, we have our version of bootstrap with the patches applied. We added this to the test image.

But that implies we maintain our own bootstrap..which we want to avoid. 

We'd also like to avoid the need to clone the kubernetes/test-infra repo to run our tests.